lib/v2/screens/profile/components/profile_bottom.dart

```diff
@@ -165,26 +166,29 @@ class ProfileBottom extends StatelessWidget {
           ),
           Padding(
             padding: const EdgeInsets.only(top: 16.0),
-            child: Container(
-              decoration: BoxDecoration(
-                color: AppColors.jungle,
-                borderRadius: BorderRadius.circular(12.0),
-              ),
-              child: ListTile(
-                leading: Icon(Icons.settings_outlined, color: Colors.white),
-                title: Text(
-                  'Settings'.i18n,
-                  style: Theme.of(context).textTheme.subtitle2,
+            child: InkWell(
```

Comment:
this inkwell is missing ink
![image](https://user-images.githubusercontent.com/8041901/108120714-b7aa3c80-706f-11eb-9300-dd22d392ee84.png)
Look at the ripple, it doesnt look good on the corners

Reply:
I see, I did'nt notice.